### PR TITLE
Add Sysbox-EE EOL announcement.

### PR DESCRIPTION
**Pull request by ctalledo**
Signed-off-by: Cesar Talledo <cesar.talledo@docker.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,30 @@
...
-We are humbled and excited to announce that Nestybox is now officially part of
-Docker, Inc! Docker is an excellent home for Sysbox, and this will accelerate
-innovation of Sysbox to advance container isolation and workloads.
+After the acquisition however, Sysbox Enterprise is no longer offered as a
+standalone product but will instead be incorporated into [Docker Desktop][docker-desktop].
+As this may not suite all users, Docker plans to port some Sysbox-EE features to
+Sysbox-CE so users can replace the former with the latter. Please refer to
+[this announcement][sysbox-ee] for further info.
+
+At this time, **no new licenses of Sysbox-EE are sold and no license renewals will
```
Comment (rodnymolina) [inline README.md:18]: s / "are sold" / "will be sold"

File: README.md [hunk 2]
```diff
@@ -2,22 +2,30 @@
...
-Hi everyone, this is Cesar & Rodny, co-founders of [Nestybox](https://www.nestybox.com).
+Prior to the [acquisition by Docker][docker-acquisition] on 05/2022, Nestybox
+offered Sysbox Enterprise as an enhanced version of Sysbox (e.g., more security,
+more workloads, and official support). It was a paid, yearly license agreement
+sold at $400 per host.
 
-We are humbled and excited to announce that Nestybox is now officially part of
-Docker, Inc! Docker is an excellent home for Sysbox, and this will accelerate
-innovation of Sysbox to advance container isolation and workloads.
+After the acquisition however, Sysbox Enterprise is no longer offered as a
```
Comment (rodnymolina) [inline README.md:12]: s / "is no longer" / "will be no longer"

**Review by rodnymolina — APPROVED** [review]
LG

File: README.md (hunk 1 above)
Comment (ctalledo) [reply README.md:18]: Fixed, thanks.

File: README.md (hunk 2 above)
Comment (ctalledo) [reply README.md:12]: Fixed.

**Pull request closed without merging**